jaxlib/gpu/triton.cc

```diff
@@ -132,6 +132,17 @@ NB_MODULE(_triton, m) {
           return major * 10 + minor;
         }));
 
+  m.def("get_arch_details",
+        ValueOrThrowWrapper([](int device)->absl::StatusOr<absl::string_view>{
+          #ifdef JAX_GPU_HIP
+            hipDeviceProp_t prop;
+            hipGetDeviceProperties(&prop, 0);
+            return prop.gcnArchName; 
+          #else
+            return "Not supported";
```

Comment:
I think you probably need something like `absl::UnimplementedError("Not a HIP GPU")`.

Reply:
Done